facia-purger/src/main/scala/com/gu/purge/facia/ParseS3Path.scala

```diff
@@ -0,0 +1,20 @@
+package com.gu.purge.facia
+
+import org.parboiled2.{ Parser, ParserInput, Rule1 }
+
+import scala.util.Success
+
+// /aws-frontend-store/CODE/frontsapi/pressed/live/au/fapi/pressed.json
+class ParseS3Path(stage: String, val input: ParserInput) extends Parser {
+  def prefix = rule { s"$stage/frontsapi/pressed/live/" }
```

Comment:
I added type annotation automatically for `front`... 😀 

`Rule[HNil, _root_.shapeless.ops.hlist.Prepend.Aux[_root_.shapeless.HNil, _root_.shapeless.::[_root_.scala.Predef.String, _root_.shapeless.HNil], _root_.shapeless.::[_root_.scala.Predef.String, _root_.shapeless.HNil]]#Out]`

Slightly better with unnecessary prefixes removed:

`Rule[HNil, Aux[HNil,::[String, HNil], ::[String, HNil]]#Out]`